Mathlib/GroupTheory/Abelianization.lean

```diff
@@ -77,6 +77,21 @@ theorem commutator_centralizer_commutator_le_center :
   rw [Subgroup.commutator_comm, Subgroup.commutator_eq_bot_iff_le_centralizer]
   exact Set.centralizer_subset (Subgroup.commutator_mono le_top le_top)
 
+/-- If g is conjugate to g ^ 2, then g is a commutator -/
+theorem mem_commutatorSet_of_isConj_sq {g : G} (hg : IsConj g (g ^ 2)) :
+    g ∈ commutatorSet G := by
+  obtain ⟨h, hg⟩ := hg
+  use ↑h; use g
+  rw [commutatorElement_def, hg]
+  simp only [IsUnit.mul_inv_cancel_right, Units.isUnit, mul_inv_eq_iff_eq_mul, pow_two]
+
+theorem Subgroup.map_commutator_eq {H : Type*} [Group H] (f : G →* H) :
```

Comment:
The name start with `Subgroup` but there is no subgroup in the theorem (I mean, there is the commutator, but no `H : Subgroup G`). Anyway not very important. (If it doesn't start with `Subgroup.` I think you can remove the `_root_.`)